Gem/Transfer/ViewsModels/PriceImpactViewModel.swift

```diff
@@ -83,9 +68,8 @@ struct PriceImpactViewModel {
         }
     }
     
-    private func calculatePriceImpact(fromPrice: Double, expectedPrice: Double) -> Double? {
-        guard fromPrice > 0 else { return nil }
-        let priceImpact = (1 - (expectedPrice / fromPrice)) * 100
-        return priceImpact
+    private func calculatePriceImpact(fromValue: Double, expectedValue: Double) -> Double? {
```

Comment:
```suggestion
    private func calculatePriceImpact(fromValue: Double, expectedValue: Double) -> Double {
```

price impact could be positive or negative?